app/package.json

```diff
@@ -29,6 +29,7 @@
     "react-konva-utils": "^0.2.0",
     "react-redux": "^7.2.6",
     "react-router-dom": "^6.4.1",
+    "react-window": "^1.8.10",
```

Comment:
Where is this used?

Reply:
It is not used. It was part of an attempt. I need to clean that up.